packages/wdio-appium-service/package.json

```diff
@@ -33,8 +33,10 @@
   "bugs": {
     "url": "https://github.com/webdriverio/webdriverio/issues"
   },
+  "devDependencies": {
+    "appium": "^1.13.0"
```

Comment:
I am not sure if we need Appium whatsoever because in dev I don't really need it too

Reply:
let me check and remove if possible

Reply:
there were some changes required in test but I've removed appium.

Reply:
there were some changes required in test but I've removed appium.